golangServer/api/search.go

```diff
@@ -60,21 +60,25 @@ func search(event *Oracle.OracleSearch) {
 		config.OracleContract.GetRsp(tps, event.ReqID, statement, []byte{}, event.CallBack, event.Sender, info)
 		return
 	}
+	k := 1
 	for _, q := range querys {
-		results, err := dbC.Search(context.Background(), q)
+		data += "\nQuery:" + string(k) + "\n"
```

Comment:
Instead of a text format, could you build up a structure and serialize it to dag-json?

e.g. something that looks like:

```
QueryResults {
  results: []{
    status: bool
    records: []Record,
    error: String | nil
  }
}
```

This way it'll be easier to parse out

Reply:
Good idea, it's already been implemented